mypy.ini

```diff
@@ -0,0 +1,19 @@
+
+[mypy]
+warn_unused_ignores = False
+ignore_missing_imports = True
+
+[mypy-jose.backends.pycrypto_backend]
+ignore_missing_imports = True
+
+[mypy-cryptography.*]
+ignore_missing_imports = True
+
+[mypy-rsa.*]
+ignore_missing_imports = True
+
+[mypy-ecdsa.*]
+ignore_missing_imports = True
+
+[mypy-pyasn1.*]
+ignore_missing_imports = True
```

Comment:
[Mypy 0.900](http://mypy-lang.blogspot.com/2021/06/mypy-0900-released.html) supports `pyproject.toml` for configuration. Might be worth using it as one less file to worry about, plus `black` and `isort` are configuration from there both.

Reply:
I would also add this configuration in `pyproject.toml` or `setup.cfg`, but since there is a separate `.flake8` file, I decided to make it separate. Perhaps it also should be reduced.

Reply:
I will be moving/converting the contents of the `.flake8` file to `pyproject.toml` at some point, so let's just start using `pyproject.toml` for all further configuration and treat `.flake8` as a legacy file.

Reply:
fixed